website/docs/table-design/table-types/pk-table/merge-engines/versioned.md

```diff
@@ -5,4 +5,66 @@ sidebar_position: 3
 
 # Versioned Merge Engine
 
-TODO: Fill me #459
\ No newline at end of file
+By specifying `'table.merge-engine' = 'versioned'`, users can update data based on the configured version column. Updates will be carried out when the latest value of the specified field is greater than the stored value. If it is less than or empty, no update will be made.
+
```

Comment:
I'd like to suggest to add a statement to explain how the feature is useful just like `first-row` merge page does, maybe something like:

```
This feature is particularly valuable for replacing [Deduplication](https://nightlies.apache.org/flink/flink-docs-release-1.20/docs/dev/table/sql/queries/deduplication/) transformations in streaming computations, reducing complexity and improving overall efficiency.
```